src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -117,6 +119,36 @@ export const prerender = true;
 </html>
 ```
 
+```astro title="src/pages/mypage.mdx" {2}
+---
+layout: '../layouts/markdown.astro'
+title: 'My page'
+---
+export const prerender = true;
+
+# This is my page
+```
+
+```astro title="src/pages/myendpoint.ts" {2}
+export const prerender = true;
+
+export async function get() {
+  return {
+    body: JSON.stringify({ message: `This is my endpoint` }),
+  };
+}
+```
```

Comment:
```suggestion
```
I suggest we don't include *both* a `.js` and `.ts` example, especially since they are so similar. In the docs, we typically only include `.js` and it's assumed that people will apply these examples to `.ts`.